### PR TITLE
Sv/ajax preload gif

### PR DESCRIPTION
**Pull request by interstellaridea**
@thomascarli  js for DOM results are done! 

i cached more DOM elements that were being used more than once and added more to the preLoader function that helps clear the last results on a new search 

lemme get that 1+ ??? :)

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -19,7 +19,21 @@ var lat = "";
 var lon = "";
 
 function handle_blank_input_error() {
-  $('.error-text').text("Protip: No one likes you. Search for something asshat.");
+  $error_text.text("Protip: No one likes you. Search for something asshat.");
```
Comment (thomascarli) [inline app/assets/javascripts/application.js]: love it!

File: app/views/locations/index.html.erb [hunk 2]
```diff
@@ -7,6 +7,11 @@
 
 	<br>
 
+	<div class="loading-container">
+		<p>Please wait while your information loads.</p>
```
Comment (thomascarli) [inline app/views/locations/index.html.erb]: we should spice this up

Comment (thomascarli) [inline app/views/locations/index.html.erb]: any ideas for something edgier?

Comment (interstellaridea) [inline app/views/locations/index.html.erb]: yea.. hmm maybe like.. "Lookin' for your shit, fuckin wait..."

Comment (interstellaridea) [inline app/views/locations/index.html.erb]: i also would like to use a better/ larger gif as well

Comment (thomascarli) [inline app/views/locations/index.html.erb]: yasss! <3 it.  Let's take care of that in a separate branch then, this one loooks good +1

**Pull request merged**